tests/clusterclient_async_sequence.c

```diff
@@ -0,0 +1,132 @@
+/*
+ * This program connects to a Redis node and then reads commands from stdin, such
+ * as "SET foo bar", one per line and prints the results to stdout.
+ *
+ * The behaviour is similar to that of clusterclient_async.c, but it sends the
+ * next command after receiving a reply from the previous command.
+ * It is still possible to send multiple commands at once like in
+ # clusterclient_async.c using following action commands:
+ #
+ # !async - Send multiple commands and wait for their responses before sending next.
+ #          Will send all following commands until EOF or the command `!sync`
```

Comment:
This doesn't sound right: "wait for their response before sending next"... Shouldn't it be "wait for their responses after all commands are sent" or something like that?